### PR TITLE
adding some templates and star wars analysis demo

### PR DESCRIPTION
**Pull request by seth127**
This is just to show the proposed method. 

Volunteers would:
* Pick a question to answer
  * Register that they're working on it in whatever project management system we decide on
  * We could use Github issues, though they're pretty basic
* Clone this repo and make a branch
* Do your analysis in whatever language you please
  * Put it in its own directory in `analysis`
  * Check in any code there as you work
* When finished, make a PR into the `main` branch and assign one of the project leads to review it
  * Probably put a link to the PR in the "ticket" or whatever in our project management system
* When the PR is approved and merged, render the content as HTML or PDF and make a PR to put that in `https://github.com/code-for-charlottesville/code-for-charlottesville.github.io/LAJC-expungement/`
* Once that is approved and merged, the content will be live for sharing with LAJC
  * We will have a link like `https://code-for-charlottesville.github.io/LAJC-expungement/my-analysis` that we can share
  * That link can be put in the ticket too, for record keeping

An example of this final step is here: https://github.com/code-for-charlottesville/code-for-charlottesville.github.io/pull/1

### CODE REVIEW

**Comment by seth127** [comment]
PS/ @jkropko if we're gonna do this you should definitely enable branch protection on `main` so that no one can push/merge anything to that branch until it has been approved by a reviewer. Same thing on [the github.io repo](https://github.com/code-for-charlottesville/code-for-charlottesville.github.io).

File: analysis/aa-fake-star-wars-question/star-wars-example.Rmd [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# Introduction
+
+This is just some fake data to show where to put things. Just showing some of the demo stuff from [dplyr](https://dplyr.tidyverse.org/#usage) so that we have something to render.
+
+**The idea is to put each analysis question in it's own repo.** This is an analysis of the question: "Who are the characters from Tatooine, Alderaan, and Kashyyyk?"
```
Comment (benfrye) [inline analysis/aa-fake-star-wars-question/star-wars-example.Rmd:16]: @seth127 Are you suggesting we have a different repo on our end for each question, or just when a volunteer wants to work on a question, they fork this repo into their own repo while they're working on it?

Comment (seth127) [reply analysis/aa-fake-star-wars-question/star-wars-example.Rmd:16]: Just use this repo, but I wasn't even thinking the volunteer would need to fork, just clone this repo and make a branch. If we put branch protection on `main` for this repo then no one will be able to push to that branch unless they 

* make their own branch
* make a PR into `main`
* get someone to review and approve that PR

## Rationale: to branch or to fork?
[This post](https://stackoverflow.com/a/34343080/3743213) lays out a pretty good pros and cons for branching vs. forking. To me, the most relevant are:

**Branching pro:** simpler and keeps all the work in one place.

**Branching con:** need to add each contributor as a collaborator to the repo.

**Forking cons:** to me, all the ones on that post boil down to "it's more complicated" in various ways.

Generally speaking, I think of forks being good for open source projects when you have no idea who the contributors are or might be, and branches are easier for when you have a known team (i.e. within a company or organization) who will be making the contributions.

That concern aside, [this post](https://softwareengineering.stackexchange.com/a/237506) also makes this conceptual argument:

> The answer to the question "should I clone or fork" is exactly the same as the answer to this question "do I want my own personal version of this project?" yes = fork, no = clone the repository.

In that light, the answer for us is obviously branch. ([This lengthier post](https://www.pluralsight.com/blog/software-development/the-definitive-guide-to-forks-and-branches-in-git) says a similar thing in more detail.)

@benfrye Lemme know if you have thoughts on all of that. I don't know how much burden you think it is to need to add everyone as a contributor before they can PR, but that seems like the only real downside of branching to me. And naively, I wouldn't think it's too much of a burden. They can clone it and do work without us adding them, but to push back and make the PR they'd need to be added.

Comment (benfrye) [reply analysis/aa-fake-star-wars-question/star-wars-example.Rmd:16]: Yea I think the collaborator piece is the main reason we've traditionally done the forking path. I would be down to try the branching route, but I can foresee a lot of time spent just getting folks added as contributors each hackathon. Whereas with forking you can just point them to the repo and say "Get goin'".

Comment (seth127) [reply analysis/aa-fake-star-wars-question/star-wars-example.Rmd:16]: That makes sense. I think there would be some initial burden to get people registered, but that could even be done beforehand via slack or email. Then after that there shouldn't be too many new people each time, right? I don't know.

Honestly, either way will work and if you guys have tried the forking strategy and feel better about that then that's totally cool.

Comment (seth127) [reply analysis/aa-fake-star-wars-question/star-wars-example.Rmd:16]: Oh and either way, this is going to rely on volunteers having some basic knowledge of git (and a github account) which is probably _not_ the case for a lot of the student volunteers. I'm sure you guys have dealt with that in the past, but lemme know if you want me to lead or help with some kind of "basic git bootcamp" to get people set up and comfortable cloning and committing and pushing. 

I stumbled onto some content that you already have [here](https://github.com/code-for-charlottesville/LAJC-Wage-Theft-Data#how-to-use-github) that's a good place to start, although I guess there are probably hundreds of "gitting started" tutorials out there.

**Review by benfrye — APPROVED** [review]
Want to pull these in @seth127 Seem like good templates to me.

**Pull request merged**